modules/deploying-node-tuning-operator.adoc

```diff
@@ -2,19 +2,24 @@
 //
 // * scalability_and_performance/using-node-tuning-operator.adoc
 
-
 [id='deploying-the-node-tuning-operator-{context}']
-= Deploying the node tuning operator
-
-To deploy the operator, apply the operator's manifest to the desired namespace
-in the cluster, create an application instance, and update an application.
+= Deploying the node tuning Operator
 
-.Prerequisites
+The Operator is deployed by applying the manifests in the Operator's `/manifests`
+directory. It automatically creates a default deployment and custom resource
+(CR) for the tuned daemons. The following shows the default CR created by the
+Operator on a cluster with the Operator installed:
 
-* List of prerequisites go here
+----
+$ oc get Tuned
+NAME      AGE
+default   1h
+----
 
 .Procedure
```

Comment:
I feel like we can tighten this procedure up.

Reply:
What do you mean by tighten up?

Reply:
@jmencak We list these steps, but they are not very explicit:

1. Apply the Operator’s manifest to the desired namespace in the cluster.
2. Create an application instance.
3. Update an application.

Are there commands that need to run to accomplish these steps? How would the user do these tasks?

Reply:
I agree, but this hasn't been finalized, that's why it is as it is.  The user shouldn't need to deploy the operator if it is installed by CVO.  The user should only be concerned by creating a CR if he/she needs it.